### DISCUSSION TITLE
Store Conversations and Messages

### DISCUSSION THREAD

**Discussion by AndreaLeto00** [danny-avila/LibreChat#2022] [Q&A]
Hi all, 

I was wondering if there was a possibility to save users' conversations and messages to mongoDB permanently, I am aware that at the moment it already does, but when a user deletes the conversation from the left side menu the conversation and attached messages disappear from Mongo.|

Thanks, hope you can help me :)

**Comment by martinounap** [comment]
Hi, and I would love to extend this feature to be able to configure how log messages are kept on database. So we would be abled to define that all the messages older than X days/month/year would be deleted. And we could add this to our terms. Right now it seems, that if user does not delete mesaages, they will be stored infinitely. 
@danny-avila Should we covert it to fetaure request?
